File: contacto-model-fix/domain/model/Contacto.kt
package com.uth.cloudcontacts.domain.model

data class Contacto(
    val id: Int,
    val nombre: String,
    val telefono: String,
    val direccion: String,
    val latitud: Double,
    val longitud: Double,
    val firmaBase64: String?,
    val imagenBase64: String?,
    val fechaCreacion: String,
    val usuarioId: Int,
    val status: Int = 1
)

File: api/src/ExamenApi.Domain/Entities/Contacto.cs
namespace ExamenApi.Domain.Entities
{
    public class Contacto
    {
        public int Id { get; set; }
        public string Nombre { get; set; } = string.Empty;
        public string Telefono { get; set; } = string.Empty;
        public string Direccion { get; set; } = string.Empty;
        public double Latitud { get; set; }
        public double Longitud { get; set; }
        public string? FirmaBase64 { get; set; }
        public string? ImagenBase64 { get; set; }
        public DateTime FechaCreacion { get; set; } = DateTime.UtcNow;
        public int Status { get; set; } = 1; // 1 = Activo, 0 = Eliminado

        public int UsuarioId { get; set; }
        public Usuario? Usuario { get; set; }
    }
}
